frontend/src/components/views/MapView.jsx

```diff
@@ -28,6 +28,10 @@ export default function MapView({assets, gameMap, overrides, mouseInput})
                 e.stopPropagation();
                 mouseInput.triggerHover(tile.x, tile.y, tile.z);
             };
+        } else {
```

Comment:
Seems like this is to make the new tile preview transparent? Mind adding a comment to this one and the previous one above?

```suggestion
        } else {
            // Make preview for new tiles semi-transparent
```